### DISCUSSION TITLE
is there any chance of using it with a python built application

### DISCUSSION THREAD

**Discussion by Jaswanth28** [pocketbase/pocketbase#851] [Q&A]
is there any way of integrating it with python and us it via web across the users distributed among many places

**Comment by ganigeorgiev** [comment]
There is no official Python SDK, but you can always use the REST API. The API documentation could be found under the "Web API" section in https://pocketbase.io/docs/api-records/.

_Please note that with v0.8 (this weekend) there would be a slight breaking change to the users API in relation to #376._

**Comment by perfalle** [comment]
I am using this unofficial client https://github.com/vaphes/pocketbase and it seems to work well so far.

**Reply by benallfree** [reply]
Thanks for sharing this, I added it to https://github.com/benallfree/awesome-pocketbase
